Docs/Transformations: Add documentation about Binary operations in Add field from calculation (#29511)
grafana/grafana: docs/sources/panels/transformations/types-options.md

Grafana comes with the following transformations:

- [Reduce](#reduce)
- [Filter by name](#filter-by-name)
- [Filter data by query](#filter-data-by-query)
- [Filter data by value](#filter-data-by-value)
- [Organize fields](#organize-fields)
- [Outer join](#join-by-field-outer-join)
- [Series to rows](#series-to-rows)
- [Add field from calculation](#add-field-from-calculation)
- [Labels to fields](#labels-to-fields)
- [Group by](#group-by)
- [Merge](#merge)

> 

Author: Just formatting change based on our linter settings, no new content.

  - **Reduce row -** Apply selected calculation on each row of selected fields independently.
  - **Binary option -** Apply basic math operation(sum, multiply, etc) on values in a single row from two selected fields.
- **Field name -** Select the names of fields you want to use in the calculation for the new field.

> - **Calculation -** If **Reduce row** mode is chosen, Calculation field is provided to create the new field. Click in the field to see a list of calculation choices. For information about available calculations, refer to the [Calculation list]({{< relref "../calculations-list.md" >}}).

Reviewer: ```suggestion
- **Calculation -** If you select **Reduce row** mode, then the **Calculation** field appears. Click in the field to see a list of calculation choices you can use to create the new field. For information about available calculations, refer to the [Calculation list]({{< relref "../calculations-list.md" >}}).
```

Reviewer: Please make sure my edit is correct in terms of what the feature is doing.

Author: Yes, very good edits. It makes sense and makes everything clear.

> - **Operation -** If **Binary option** mode is chosen, Operation field is provided to do basic math operations on values in a single row from two selected fields. You can also use numerical values for binary operations.

Reviewer: ```suggestion
- **Operation -** If you select **Binary option** mode, then the **Operation** field appears. This field allows you  to do basic math operations on values in a single row from two selected fields. You can also use numerical values for binary operations.
```

> - **Operation -** If you select **Binary option** mode, then the **Operation** field appears. This field allows you  to do basic math operations on values in a single row from two selected fields. You can also use numerical values for binary operations.

Author: ```suggestion
- **Operation -** If you select **Binary option** mode, then the **Operation** fields appear. These fields allow you  to do basic math operations on values in a single row from two selected fields. You can also use numerical values for binary operations.
```

> **Note:** This transformation is only available in Grafana 7.2+.

This transformation groups the data by a specified field (column) value and processes calculations on each group. Click to see a list of calculation choices. For information about available calculations, refer to the [List of calculations]({{< relref "../calculations-list.md" >}}).

> 

Author: Just formatting change based on our linter settings, no new content.

This transformation goes in two steps. First you specify one or multiple fields to group the data by. This will group all the same values of those fields together, as if you sorted them. For instance if we group by the Server ID field, then it would group the data this way:

| Time                | Server ID      | CPU Temperature | Server Status |
| ------------------- | -------------- | --------------- | ------------- |
| 2020-07-07 11:34:20 | **server 1**   | 80              | Shutdown      |
| 2020-07-07 09:28:06 | **server 1**   | 80              | OK            |
| 2020-07-07 09:23:07 | **server 1**   | 86              | OK            |
| 2020-07-07 10:32:20 | server 2       | 90              | Overload      |
| 2020-07-07 09:30:05 | server 2       | 88              | OK            |
| 2020-07-07 09:25:05 | server 2       | 88              | OK            |
| 2020-07-07 11:34:20 | **_server 3_** | 62              | OK            |
| 2020-07-07 10:31:22 | **_server 3_** | 55              | OK            |
| 2020-07-07 09:30:57 | **_server 3_** | 62              | Rebooting     |

> 

Author: Just formatting change based on our linter settings, no new content.

> **Note:** This transformation is only available in Grafana 7.1+.

Use this transformation to combine the result from multiple time series data queries into one single result. This is helpful when using the table panel visualization.

> 

Author: Just formatting change based on our linter settings, no new content.

## Filter by value

> This transformation allows you to filter your data directly in Grafana and remove some data points from your query result. You have the option to include or exclude data that match one or more conditions you define. The conditions are applied on a selected field.

Author: Just formatting change based on our linter settings, no new content.

Consider the following data set:

| Time                | Temperature | Altitude |
| ------------------- | ----------- | -------- |
| 2020-07-07 11:34:23 | 32          | 101      |
| 2020-07-07 11:34:22 | 28          | 125      |
| 2020-07-07 11:34:21 | 26          | 110      |
| 2020-07-07 11:34:20 | 23          | 98       |
| 2020-07-07 10:32:24 | 31          | 95       |
| 2020-07-07 10:31:22 | 20          | 85       |
| 2020-07-07 09:30:57 | 19          | 101      |

> 

Author: Just formatting change based on our linter settings, no new content.

And you will get the following result, where only the temperatures below 30°C are included:

| Time                | Temperature | Altitude |
| ------------------- | ----------- | -------- |
| 2020-07-07 11:34:22 | 28          | 125      |
| 2020-07-07 11:34:21 | 26          | 110      |
| 2020-07-07 11:34:20 | 23          | 98       |
| 2020-07-07 10:31:22 | 20          | 85       |
| 2020-07-07 09:30:57 | 19          | 101      |

> 

Author: Just formatting change based on our linter settings, no new content.
